components/menu-breadcrumbs-sd.tpl

```diff
@@ -3,9 +3,8 @@
 
 <ul class="menu menu-horizontal menu-public menu-breadcrumbs">
   {%- for listItem in breadcrumbsObj.itemListElement %}
-    {%- assign pageUrl = page.url | remove_first: "/" -%}
     <span class="menu-separator">/</span>
-    <li class="selected menu-item {% if pageUrl == listItem.item.id %} current{% endif %}">
+    <li class="selected menu-item {% if forloop.last %} current{% endif %}">
```

Comment:
Here the `selected` class should be removed as well.

A small code-style improvement can be done here as well. If the `current` class isn't included, there is excess whitespace after the `menu-item` class. Remove one of the spaces (either before `current` or after `menu-item`).